ansible/playbooks/AdoptOpenJDK_AIX_Playbook/roles/ant/tasks/main.yml

```diff
@@ -18,11 +18,23 @@
         msg: "Ant installed, skipping download and installation"
       when: ant.stat.islnk is defined
 
-    - name: Download and extract ant
+    - name: Download Apache Ant
+      get_url:
+        url: https://archive.apache.org/dist/ant/binaries/apache-ant-1.9.9-bin.zip
```

Comment:
Do we want the latest version of Ant?

Reply:
In order to avoid any risk of inconsistency between systems I would suggest it is preferable to lock it and do upgrades of this tool in a more controlled manner. We fix the versions in many other places and if we wish to change the policy on it (similar to what we're trying to do with the compilers) we should raise and discuss as a separate issue to cover all instances of this in all platform playbooks.